inject/src/main/java/io/micronaut/inject/annotation/AnnotationMetadataSupport.java

```diff
@@ -94,7 +94,7 @@
      */
     @UsedByGeneratedCode
     public static Map<String, Object> getDefaultValues(String annotation) {
-        return ANNOTATION_DEFAULTS.computeIfAbsent(annotation, s -> Collections.emptyMap());
+        return ANNOTATION_DEFAULTS.getOrDefault(annotation, Collections.emptyMap());
```

Comment:
This doesn't look like it was correct, inserting an immutable empty map where the value can be modified and some of the code checked if default values are present.